advanced_functionality/autogluon-sagemaker-pipeline/sagemaker-pipelines-project.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<p>Typo: <code>paradigmsfor</code> -&gt; <code>paradigms for</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3450/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done!